### PR TITLE
Easier installation and setup instructions for new users

### PR DESCRIPTION
**Pull request by martinnaughton**
Just tried vue-fullpage.js today but it took a bit of looking around to figure out the setup instructions. i thought these instructions would make it easier for any other new person wanting to set it up

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,6 +28,28 @@ bower install vue-fullpage.js
...
+
+Add the following links to your index.html head element in your project. These are the dependencies for fullpage.js. Update the versions as you require.
+```shell
+<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/fullPage.js/2.9.5/jquery.fullpage.min.js"></script>
+<link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/fullPage.js/2.9.5/jquery.fullpage.min.css">
+```
+
+setup the component to be used in the project in src/main.js
+```shell
```
Comment (funkyfuture) [inline README.md]:
```suggestion
```js
```

File: README.md [hunk 2]
```diff
@@ -34,6 +28,28 @@ bower install vue-fullpage.js
...
+// With bower
+bower install fullpage.js
+
+// With npm
+npm install fullpage.js
+```
+
+Add the following links to your index.html head element in your project. These are the dependencies for fullpage.js. Update the versions as you require.
+```shell
```
Comment (funkyfuture) [inline README.md]:
```suggestion
```html
```

File: README.md [hunk 3]
```diff
@@ -34,6 +28,28 @@ bower install vue-fullpage.js
 npm install vue-fullpage.js
 ```
 
+### Dependencies
+```shell
+// With bower
```
Comment (funkyfuture) [inline README.md]:
```suggestion
# With bower
```

File: README.md [hunk 4]
```diff
@@ -34,6 +28,28 @@ bower install vue-fullpage.js
...
+bower install fullpage.js
+
+// With npm
```
Comment (funkyfuture) [inline README.md]:
```suggestion
# With npm
```

**Comment by dobromir-hristov** [comment]
Unnecessary.

**Pull request closed without merging**